tests/staticfiles_tests/test_liveserver.py

```diff
@@ -13,10 +14,10 @@
 
 TEST_ROOT = os.path.dirname(__file__)
 TEST_SETTINGS = {
-    'MEDIA_URL': 'media/',
-    'STATIC_URL': 'static/',
-    'MEDIA_ROOT': os.path.join(TEST_ROOT, 'project', 'site_media', 'media'),
-    'STATIC_ROOT': os.path.join(TEST_ROOT, 'project', 'site_media', 'static'),
+    "MEDIA_URL": "media/",
+    "STATIC_URL": "static/",
+    "MEDIA_ROOT": os.path.join(TEST_ROOT, "project", "site_media", "media"),
+    "STATIC_ROOT": os.path.join(TEST_ROOT, "project", "site_media", "static"),
```

Comment:
I'm not sure why these were changed. Current code style is single quotes. Below also,